python/lsst/ts/hexrotcomm/base_csc.py

```diff
@@ -93,7 +101,7 @@ def make_state_transition_dict():
     }
 
     # compute transitions from non-FAULT to all other states
-    state_transition_dict = dict()
+    state_transition_dict: dict = dict()
```

Comment:
If possible, please specify the dict contents. Feel free to leave this for a future ticket.

Reply:
This is the return value in the commit: [4f2c704](https://github.com/lsst-ts/ts_hexrotcomm/pull/69/commits/4f2c704e3a773371b0ab61a92cfae6b9acd21c33). Thanks!